src/arch/riscv/isa/decoder.isa

```diff
@@ -3388,6 +3521,22 @@ decode QUADRANT default Unknown::unknown() {
                         res = int_rounding<__uint128_t>(res, 0 /* TODO */, 1);
                         Vd_vi[i] = res >> 1;
                     }}, OPMVX, VectorDummyOp);
+                    0x0e: vslide1up_vx({{
+                        // vd[0]=x[rs1], vd[i+1] = vs2[i]
+                        if (isFirstMicroop() && i == 0) {
+                            Vd_vu[i] = Rs1_vu;
+                        } else {
+                            Vd_vu[i] = Vs2_vu[i - 1];
+                        }
+                    }}, OPMVX, VectorDummyOp);
```

Comment:
The implementation of `v[f]slide[up|down].v[x|i]` cannot run correctly on O3CPU, since the Vs2s in different uops are not in a contiguous memory space. Let's reconsider how to implement these instructions.

Reply:
Do you guys have an implementation in flight? 
As a temporary measure I've pushed a commit that fails if the `v[f]slide[up|down].v[x|I]` have multiple micro ops.
I can drop these instructions altogether if you'd like instead. Let me know.

Reply:
Done. Pushed.